src/pkg/cli/client/byoc/aws/byoc.go

```diff
@@ -329,9 +329,9 @@ func (b *ByocAws) environment() map[string]string {
 	}
 }
 
-func (b *ByocAws) runCdCommand(ctx context.Context, behavior defangv1.Behavior, cmd ...string) (ecs.TaskArn, error) {
+func (b *ByocAws) runCdCommand(ctx context.Context, mode defangv1.DeploymentMode, cmd ...string) (ecs.TaskArn, error) {
 	env := b.environment()
-	env["DEFANG_BEHAVIOR"] = strings.ToLower(behavior.String())
+	env["DEFANG_MODE"] = strings.ToLower(mode.String())
```

Comment:
nit: the ToLower() is redundant as mode.String() always returns a lower case value.